gex/Services/Db/UserStats/BarUserSkillDb.cs

```diff
@@ -70,5 +70,31 @@ public async Task<List<BarUserSkill>> GetByUserID(long userID, CancellationToken
             ))).ToList();
         }
 
+        /// <summary>
+        ///     get the <see cref="BarUserSkill"/> entries for multiple users
+        /// </summary>
+        /// <param name="userIDs">List of user IDs to get skills for</param>
+        /// <param name="cancel">cancellation token</param>
+        /// <returns>
+        ///     a dictionary mapping user IDs to lists of <see cref="BarUserSkill"/>s
+        /// </returns>
+        public async Task<Dictionary<long, List<BarUserSkill>>> GetByUserIDs(List<long> userIDs, CancellationToken cancel) {
```

Comment:
strictly speaking we don't need a dictionary to return this, as `BarUserSkill` already has a `UserID` field